sts_ocm_admin_permission_policy.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:GetRole",
+        "iam:CreateRole",
+        "iam:PutRolePermissionsBoundary",
+        "iam:TagRole",
+        "iam:ListPolicies",
+        "iam:ListPolicyTags",
+        "iam:AttachRolePolicy",
+        "iam:GetOpenIDConnectProvider",
+        "iam:CreateOpenIDConnectProvider",
+        "iam:TagOpenIDConnectProvider",
+        "iam:GetPolicy"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
```

Comment:
These JSON resource files are here: https://github.com/openshift/managed-cluster-config/tree/master/resources/sts/4.10
Can they be embedded instead?